data/tools/wesnoth/wmltools3.py

```diff
@@ -145,7 +145,7 @@ def comma_split(csstring, list=None, strip="r"):
     # lstrip() only, the other to warn about trailing whitespace.
     if 'w' in strip:
         for item in vallist:
-            if re.search('\s$', item):
+            if re.search(r'\s$', item):
```

Comment:
Maybe change the single quotes to double quotes to make it consistent with the rest of the code?